importer/loader.py

```diff
@@ -7,8 +8,14 @@ class SpreadsheetLoader:
 
     def __init__(self, file):
         self._file = file
-        self._workbook = xlrd.open_workbook(self._file)
-        self._sheet = self._workbook.sheet_by_index(0)
+        try:
+            self._workbook = openpyxl.load_workbook(self._file)
+            self._sheet = self._workbook.worksheets[0]
+            self._format = 'xlsx'
+        except:
+            self._workbook = xlrd.open_workbook(self._file)
+            self._sheet = self._workbook.sheet_by_index(0)
+            self._format = 'xls'
 
     def load(self):
```

Comment:
If this is only going to be used for loading xls, could this function be renamed to load_xls so it is inline with the naming of the other function